### DISCUSSION TITLE
Support for multiple accounts

### DISCUSSION THREAD

**Discussion by leandrobrunner** [community/community#28181] [Other Feature Feedback, Questions, & Ideas]
I want to use GitHub Mobile with multiple non-enterprise accounts.

**Comment by jfullstackdev** [comment]
but GitHub encourages only one account even with multiple added emails, so I doubt whether they will be doing that request

![image](https://user-images.githubusercontent.com/47092464/182619368-c76d89f7-ca10-48cf-8a5c-90e24d25879c.png)

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/merging-multiple-personal-accounts

**Reply by ghost** [reply]
@jdevstatic Hi! Can you help me? I have a similar question... if I add 2 emails on GitHub, in that case could I have 2 usernames on the same account?... example: codehangen@website.com, codehangen@contact.com in github.com\codehangen  as codehange1, codehangen2?

**Reply by jfullstackdev** [reply]
nope, just one username with multiple email accounts, 

what you can do is to have Orgs, remember they are just like users but designed with a team on mind, you can have Orgs without members, I'm using it to separate my works and for my GitHub Pages to have default domains that are separate from my personal account, like this 

https://github.com/devz-throwable

**Reply by ghost** [reply]
jdevstatic thank you for feedback! I'm glad with your answer!
